app/package.json

```diff
@@ -63,8 +64,8 @@
     "moment-locales-webpack-plugin": "^1.0.7",
     "react-test-renderer": "^16.4.1",
     "redux-mock-store": "^1.5.3",
+    "style-loader": "^1.0.0",
     "ts-jest": "^23.0.0",
-    "typescript": "^2.9.2",
```

Comment:
oh huh, i added typescript, then changed my mind and removed it. didn't realize it was already in here. is there an intention to move to typescript?? i'd be more than happy to incrementally add it in

Reply:
Yeah I'd like to start typing things incrementally on both the backend and the frontend. Especially in cases where the required type is super ambiguous.